### ISSUE TITLE
hentaihaven

### ISSUE THREAD

**Issue by Kadantte** [HiruNya/HentaiBot#14]
hentaihaven is back Online again :D

**Comment by HiruNya** [comment]
Yup! I'm currently a bit busy with school stuff until Wednesday, so I should be able to add support for hentaihaven by the end of next week.

**Comment by scorpionknifes** [comment]
aren't you never too busy for hentai

**Comment by Kadantte** [comment]
@HiruNya Thank's and Good luck with school stuff.

**Comment by HiruNya** [comment]
I made Pull Request #16 that should support this. If you wish to use this use the "hentaihaven" branch. I will merge it with the master branch after a new release occurs so that I can test to see if it works properly.

I will keep this issue open until it is merged.

**Comment by HiruNya** [comment]
Since HentaiHaven hasn't posted anything in 2 weeks, I haven't been able to test it. Everything seems to be working fine though so I'll probably merge this branch into master in a few days.

**Comment by Kadantte** [comment]
I think they have a plan for the website:
[The Plan](https://hentaihaven.org/the-plan).

**Comment by HiruNya** [comment]
Yeah, I saw that. I'm wondering if they will change the website to support this new legal hentai content they wish to show. If they don't touch the RSS feed, then the bot will work as normal. If they do change it, then the bot will need to be updated for it to work again.

I've changed my mind and I probably wont merge it until they upload something new because I feel like they might change the website before they start uploading again and if that happens I don't want the bot to give an error.

**Comment by Kadantte** [comment]
Yea this is what I was thinking so just let this merge in dev "hentaihaven" branch until they push some updates.

So let's let this issue open until they do.
